examples/MQ/pixelDetector/src/devices/FairMQPixelFileSinkBin.cxx

```diff
@@ -100,16 +100,18 @@ void FairMQPixelFileSinkBin::Init()
 
 bool FairMQPixelFileSinkBin::StoreData(fair::mq::Parts& parts, int /*index*/)
 {
-    if (parts.Size() == 0)
+    auto numParts = parts.Size();
+
+    if (numParts == 0)
         return true;   // probably impossible, but still check
 
     // expecting even number of parts in the form: header,data,header,data,header,data and so on...
     int nPPE = 2;   // nof parts per event
```

Comment:
I think, `nPPE` is not modified in this context? Can you add a const or even constexpr to it?

Reply:
done